Src/Support/GoogleApis.Tests/Apis/Download/MediaDownloaderTest.cs

```diff
@@ -197,36 +266,51 @@ public void DownloadAsync_Cancel()
             Subtest_Download_Chunks(MediaContent.Length - 1, false, 1);
         }
 
+        /// <summary>
+        /// Uses MediaDownloader to download the contents of a URI.
+        /// Asserts that the download succeeded and returns the resulting content as a string.
+        /// </summary>
+        /// <param name="uri">Uri to download</param>
+        /// <returns></returns>
+        private string SimpleDownload(string uri)
+        {
+            using (var service = new MockClientService())
+            {
+                var downloader = new MediaDownloader(service);
+                var outputStream = new MemoryStream();
+                var result = downloader.Download(uri, outputStream);
+
+                Assert.AreEqual(result.Status, DownloadStatus.Completed);
+                Assert.IsNull(result.Exception);
+                Assert.AreEqual(result.BytesDownloaded, outputStream.Position);
+
+                return Encoding.UTF8.GetString(outputStream.GetBuffer(), 0, (int)outputStream.Position);
+            }
+        }
+
         /// <summary>A helper test to test sync and async downloads.</summary>
         /// <param name="chunkSize">The chunk size for each part.</param>
         /// <param name="sync">Indicates if this download should be synchronously or asynchronously.</param>
         /// <param name="cancelRequest">Defines the request index to cancel the download request.</param>
-        /// <param name="downloadUri">The URI which contains the media to download.</param>
-        private void Subtest_Download_Chunks(int chunkSize, bool sync = true, int cancelRequest = 0,
-            string downloadUri = "http://www.sample.com")
+        /// <param name="target">Last component of the Uri to download</param>
+        private void Subtest_Download_Chunks(int chunkSize, bool sync = true, int cancelRequest = 0, string target = "content")

```

Comment:
Possibly rename cancelRequest to cancelChunk as there's really only one request now?